examples/experimental/dagster-airlift/dagster_airlift/core/defs_builders.py

```diff
@@ -0,0 +1,42 @@
+from typing import Sequence, Union
+
+from dagster import AssetsDefinition, AssetSpec, multi_asset
+from dagster._core.definitions.asset_key import CoercibleToAssetKey
+from dagster._core.definitions.definitions_class import Definitions
+from typing_extensions import TypeAlias
+
+from .utils import DAG_ID_TAG, TASK_ID_TAG
+
+CoercibleToAssetSpec: TypeAlias = Union[AssetSpec, CoercibleToAssetKey]
+
+
+def defs_from_task(
+    *, task_id: str, dag_id: str, assets: Sequence[CoercibleToAssetSpec]
+) -> Definitions:
+    """Construct a Dagster :py:class:`Definitions` object from a provided set of assets,
+    with a mapping to which airflow task produces those assets.
+    """
+    asset_specs = [
+        asset if isinstance(asset, AssetSpec) else AssetSpec(key=asset) for asset in assets
+    ]
```

Comment:
I'd rather see just a `specs` argument and add a `AssetSpec.from_keys` helper. Just the parameter name make to coercibility weird and is ambiguous with existing `assets` parameters that are ambiguous between `AssetsDefinition` and `List[AssetsDefinition]`

Reply:
Fine with that.